lib/LMSManagers/LMSHelpdeskManager.php

```diff
@@ -1491,11 +1491,13 @@ public function TicketChange($ticketid, array $props)
             $props['source'] = $ticket['source'];
         }
 
-        if ($ticket['priority'] != $props['priority'] && isset($props['priority'])) {
+        if ($ticket['priority'] != $props['priority']) {
+            $RT_PRIORITIES[$ticket['priority']] = empty($RT_PRIORITIES[$ticket['priority']]) ? trans('unknown') : $RT_PRIORITIES[$ticket['priority']];
+            $RT_PRIORITIES[$props['priority']] = empty($RT_PRIORITIES[$props['priority']]) ? trans('unknown') : $RT_PRIORITIES[$props['priority']];
```

Comment:
To to za prowizorka? Podmieniasz coś co jest w definitions.php?

Reply:
Czy w definitions.php deklarujemy coś co nieistnieje/nie jest określone/jestnieznane? To jest dlatego żeby nie było pustego komunikatu.

Reply:
Przeglądałem definitions.php i deklarujemy tam stany różne od null. 
Jak byś zrobił?

Reply:
A co to ma robić? Trudno odczytać z tego hacka co to miało na celu :)

Reply:
https://github.com/lmsgit/lms/pull/1993#discussion_r610733559

Reply:
No to jest warstwa prezentacji, więc nie rozumiem dlaczego mieszasz do tego $RT_PRIORITIES...

Reply:
Treść komunikatu i sama wiadomość komunikatu jest dodawana jako wiadomość do tabeli rtmessages na backendzie.
W warstwie prezentacji jest cały komunikat jako jedna zmienna...